README.md

```diff
@@ -65,15 +82,39 @@ By default, there is a `GenericProvider` that support a `SimpleProcessor` using
 The library is available as a Python package in pypi and can be installed with pip:
 `pip install circuit-maintenance-parser`
 
-## Usage
+## How to use it?
+
+The library requires two things:
+
+- The `notificationdata`: this is the data that the library will check to extract the maintenance notifications. It can be simple (only one data type and content) or from an email, with multiple data parts.
```

Comment:
```suggestion
- The `notificationdata`: this is the data that the library will check to extract the maintenance notifications. It can be simple (only one data type and content, such as an iCalendar notification) or more complex (with multiple data parts of different types, such as from an email).
```